working/wildcards/feature-specification.md

```diff
@@ -76,21 +76,21 @@ library privacy comes into play.
 
 ## Proposal
 
-### Local declarations
+### Declarations that are capable of declaring a wildcard
 
-A *local declaration* is any of:
+Any of the following kinds of declarations can declare a wildcard:
 
 *   Function parameters. This includes top-level functions, local functions,
     function expressions ("lambdas"), instance methods, static methods,
-    constructors, etc. It includes all parameter kinds: simple, field formals,
-    and function-typed formals, etc.:
+    constructors, etc. It includes all parameter kinds, excluding named
```

Comment:
I want to explicitly call this out. Not sure if this is the best place to write this. I know it's in `Initializing formals`, but it'd be nice to restate here.